addons/project/views/project_sharing_views.xml

```diff
@@ -208,7 +209,7 @@
                                     <field name="stage_id" optional="show"/>
                                     <button name="action_open_task" type="object" title="View Task" string="View Task" class="btn btn-link float-end"
                                             context="{'form_view_ref': 'project.project_sharing_project_task_view_form'}"
```

Comment:
```suggestion
                                            context="{'form_view_ref': 'project.project_sharing_project_task_view_form', 'search_view_ref': 'project.project_sharing_project_task_view_search'}"
```

if it does not work then I guess we will have to change the action returned by `action_open_task` to be sure we use the right search view for project sharing. 🙂

Reply:
precising the search view in the context works, well played :+1: